.github/workflows/release.yml

```
@@ -3,7 +3,7 @@ name: Main
 on:
     push:
         tags:
-            - "*"
+            - "v1.*"
```

I don't want to miss this when going for 2.0.
```suggestion
            - "v*"
```

---

You don't want to accidentally push a v2 beta as stable `latest` either

---

Hm. Fair enough. Let's keep it this way.